paper.tex

```diff
@@ -35,6 +35,29 @@
 \begin{document}
 \maketitle
 
+\abstract{
+  Symbolic based computer aided algebra systems (CAS) play an important role in
+  scientific computing with the earliest packages dating back into the 1970s.
+  Numerous open and proprietary software packages exist that aid users in
+  complex algebra, calculus, and other mathematical concepts that drive and
+  support both symbolic and numerical computing needs.
+  Here we present a symbolic CAS written in the Python programming language
+  that is developed by a large team of scientists, engineers, and software
+  developers in a bazaar open source development model.
+  The accessibility of the codebase and the community model allow SymPy to
+  rapidly respond to the needs of the community of users without the knowledge
```

Comment:
I know. This is what I interpreted was the "novel" thing we have with SymPy from issue #55. It is here so that we rethink the theme/story of the paper so that we can add and remove content form the current version to fit a theme.

I think that SymPy outshines many other software projects wrt to how many people we have that have made non-trivial contributions. The graph we show that gives number of commits per user as compared to other scipy stack projects is quite telling and unique.